includes/util.php

```diff
@@ -76,3 +76,454 @@ function newspack_install_activate_theme() {
+		'PKR' => __( 'Pakistani rupee', 'woocommerce' ),
+		'PLN' => __( 'Polish z&#x142;oty', 'woocommerce' ),
+		'PRB' => __( 'Transnistrian ruble', 'woocommerce' ),
+		'PYG' => __( 'Paraguayan guaran&iacute;', 'woocommerce' ),
+		'QAR' => __( 'Qatari riyal', 'woocommerce' ),
+		'RON' => __( 'Romanian leu', 'woocommerce' ),
+		'RSD' => __( 'Serbian dinar', 'woocommerce' ),
+		'RUB' => __( 'Russian ruble', 'woocommerce' ),
+		'RWF' => __( 'Rwandan franc', 'woocommerce' ),
+		'SAR' => __( 'Saudi riyal', 'woocommerce' ),
+		'SBD' => __( 'Solomon Islands dollar', 'woocommerce' ),
+		'SCR' => __( 'Seychellois rupee', 'woocommerce' ),
+		'SDG' => __( 'Sudanese pound', 'woocommerce' ),
+		'SEK' => __( 'Swedish krona', 'woocommerce' ),
+		'SGD' => __( 'Singapore dollar', 'woocommerce' ),
+		'SHP' => __( 'Saint Helena pound', 'woocommerce' ),
+		'SLL' => __( 'Sierra Leonean leone', 'woocommerce' ),
+		'SOS' => __( 'Somali shilling', 'woocommerce' ),
+		'SRD' => __( 'Surinamese dollar', 'woocommerce' ),
+		'SSP' => __( 'South Sudanese pound', 'woocommerce' ),
+		'STD' => __( 'S&atilde;o Tom&eacute; and Pr&iacute;ncipe dobra', 'woocommerce' ),
+		'SYP' => __( 'Syrian pound', 'woocommerce' ),
+		'SZL' => __( 'Swazi lilangeni', 'woocommerce' ),
+		'THB' => __( 'Thai baht', 'woocommerce' ),
+		'TJS' => __( 'Tajikistani somoni', 'woocommerce' ),
+		'TMT' => __( 'Turkmenistan manat', 'woocommerce' ),
+		'TND' => __( 'Tunisian dinar', 'woocommerce' ),
+		'TOP' => __( 'Tongan pa&#x2bb;anga', 'woocommerce' ),
+		'TRY' => __( 'Turkish lira', 'woocommerce' ),
+		'TTD' => __( 'Trinidad and Tobago dollar', 'woocommerce' ),
+		'TWD' => __( 'New Taiwan dollar', 'woocommerce' ),
+		'TZS' => __( 'Tanzanian shilling', 'woocommerce' ),
+		'UAH' => __( 'Ukrainian hryvnia', 'woocommerce' ),
+		'UGX' => __( 'Ugandan shilling', 'woocommerce' ),
+		'USD' => __( 'United States (US) dollar', 'woocommerce' ),
+		'UYU' => __( 'Uruguayan peso', 'woocommerce' ),
+		'UZS' => __( 'Uzbekistani som', 'woocommerce' ),
+		'VEF' => __( 'Venezuelan bol&iacute;var', 'woocommerce' ),
+		'VES' => __( 'Bol&iacute;var soberano', 'woocommerce' ),
+		'VND' => __( 'Vietnamese &#x111;&#x1ed3;ng', 'woocommerce' ),
+		'VUV' => __( 'Vanuatu vatu', 'woocommerce' ),
+		'WST' => __( 'Samoan t&#x101;l&#x101;', 'woocommerce' ),
+		'XAF' => __( 'Central African CFA franc', 'woocommerce' ),
+		'XCD' => __( 'East Caribbean dollar', 'woocommerce' ),
+		'XOF' => __( 'West African CFA franc', 'woocommerce' ),
+		'XPF' => __( 'CFP franc', 'woocommerce' ),
+		'YER' => __( 'Yemeni rial', 'woocommerce' ),
+		'ZAR' => __( 'South African rand', 'woocommerce' ),
+		'ZMW' => __( 'Zambian kwacha', 'woocommerce' ),
+	);
+	return $currencies;
+}
+
+/**
+ * Get full list of country codes. Copied from https://github.com/woocommerce/woocommerce/blob/master/includes/class-wc-countries.php
+ *
+ * @return array
+ */
+function newspack_countries() {
+	$countries = array(
+		'AF' => __( 'Afghanistan', 'woocommerce' ),
```

Comment:
Same comment as above about the textdomain.

Reply:
https://github.com/Automattic/newspack-plugin/pull/114/commits/229ca041d293900d763d9a93de3eee43e9bf4b0f